specification/schemas/invitations/InvitationResponse.json

```diff
@@ -0,0 +1,59 @@
+{
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/Invitation"
+    },
+    {
+      "required": [
+        "id",
+        "attributes",
+        "relationships",
+        "links"
+      ],
+      "properties": {
+        "attributes": {
+          "required": [
+            "email"
+          ],
+          "expires_at": {
+            "readOnly": true
+          },
+          "properties": {
+            "first_name": {
+              "type": "string",
+              "example": "Sherlock"
+            },
+            "last_name": {
+              "type": "string",
+              "example": "Holmes"
+            },
+            "email": {
+              "$ref": "#/components/schemas/Email",
+              "example": "s.holmes@holmesinvestigations.com"
+            }
```

Comment:
These properties are already defined in the Invitation schema, no need to redefine them.